### PR TITLE
Adds vk-bootstrap

### PR DESCRIPTION
**Pull request by zheng95z**
Add support for Vulkan Bootstrapping Iibrary.

### CODE REVIEW

File: packages/v/vk-bootstrap/xmake.lua [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package("vk-bootstrap")
+    set_homepage("https://github.com/charles-lunarg/vk-bootstrap")
+    set_description("Vulkan Bootstrapping Iibrary.")
+    set_license("MIT")
+
+    add_urls("https://github.com/charles-lunarg/vk-bootstrap.git")
```
Comment (waruqi) [inline packages/v/vk-bootstrap/xmake.lua:6]: please add tar urls and versions

```lua
add_urls("https://github.com/charles-lunarg/vk-bootstrap/archive/refs/tags/$(version).tar.gz",
         "https://github.com/charles-lunarg/vk-bootstrap.git")

add_versions("v0.5", "<tar.gz sha256>")
```

File: packages/v/vk-bootstrap/xmake.lua [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                set_languages("cxx14")
+                add_files("src/VkBootstrap.cpp")
+                add_headerfiles("src/VkBootstrap.h")
+                add_headerfiles("src/VkBootstrapDispatch.h")
+                add_packages("vulkan-headers")
+                if is_plat("linux") then
+                    add_syslinks("dl")
+                end
+        ]])
+        import("package.tools.xmake").install(package, {"-m release"})
```
Comment (waruqi) [inline packages/v/vk-bootstrap/xmake.lua:31]: only support release? you can remove `add_rules("mode.debug")` and only reserve mode.release

and remove `, {"-m release"}`

File: packages/v/vk-bootstrap/xmake.lua (hunk 1 above)
Comment (zheng95z) [reply packages/v/vk-bootstrap/xmake.lua]: Done.

File: packages/v/vk-bootstrap/xmake.lua (hunk 2 above)
Comment (zheng95z) [reply packages/v/vk-bootstrap/xmake.lua]: It's a mistake. Done fixing.

**Pull request merged**